src/patterns/task-list-pages/index.md.njk

```diff
@@ -75,6 +77,33 @@ For example, you could group all tasks which help users find out if your service
 
 Where possible, allow users to complete tasks in any order. This will help them plan their time and complete sections as and when they can.
 
+### Marking tasks as completed
+
+Sometimes, it’s better for the user to decide when a task is completed.
+
+This can be helpful when a task involves:
+
+* some questions that are optional
+* writing a long answer (such as in a textarea)
+* looking up information, such as details about previous jobs
+* answers that need to be checked carefully with someone else
+
+Do this by asking a radio question at the end of the task — either as the last question (if the task involves a single page) or on the [‘Check answers’ page](/patterns/check-answers/) (if the task contains multiple [question pages](/patterns/question-pages/)).
```

Comment:
```suggestion
Do this by asking a radio question at the end of the task — either as the last question (if the task involves a single page) or on the [‘Check answers’ page](/patterns/check-answers/) (if the task involves multiple [question pages](/patterns/question-pages/)).
```

Reply:
How about "if the task uses multiple question pages"?

Reply:
I'm fine with 'uses', my thinking with this comment was more about consistency within the sentence (and saying 'contains' and 'involves' to for the same thing).

How about:
```suggestion
Do this by asking a radio question at the end of the task — either as the last question (if the task is a single page) or on the [‘Check answers’ page](/patterns/check-answers/) (if the task uses multiple [question pages](/patterns/question-pages/)).
```

I think the ideal word would be "on" ("on a single page" and "on multiple pages"), but it'd create confusing duplication in the sentence.

Reply:
@calvin-lau-sig7 perfect, thanks. Have committed your suggestion. 👍